### PR TITLE
Feature/#1851 extended export csv file

### PR DESCRIPTION
**Pull request by cayop**
There are one example of csv in proposal_extended_csv_report.csv
I take care for search None fields but can be some of them.
I use test_export_extended in test_documents for to do different manual checks too.

### CODE REVIEW

File: ereuse_devicehub/resources/documents/device_row.py [hunk 1]
```diff
@@ -42,105 +70,231 @@ def __init__(self, device: d.Device) -> None:
...
+        if component is None:
+            self['{} {} Size'.format(ctype, i)] = ''
+            self['Erasure {} {}'.format(ctype, i)] = ''
+            self['Erasure {} {} Serial Number'.format(ctype, i)] = ''
+            self['Erasure {} {} Size'.format(ctype, i)] = ''
+            self['Erasure {} {} Software'.format(ctype, i)] = ''
+            self['Erasure {} {} Result'.format(ctype, i)] = ''
+            self['Erasure {} {} Type'.format(ctype, i)] = ''
+            self['Erasure {} {} Method'.format(ctype, i)] = ''
+            self['Erasure {} {} Elapsed'.format(ctype, i)] = ''
```
Comment (JNadeu) [inline ereuse_devicehub/resources/documents/device_row.py:197]: Add units on field name. 'Elapsed (hours)'.

File: tests/files/complete.export.snapshot.yaml [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+        "steps": [
+          {
+            "startTime": "2019-08-19T16:48:19.690458+00:00",
+            "type": "StepRandom",
+            "severity": "Info",
+            "endTime": "2019-08-19T17:34:22.930562+00:00"
+          }
+        ],
+        "severity": "Info",
+        "endTime": "2019-08-19T17:34:22.930959+00:00"
```
Comment (JNadeu) [inline tests/files/complete.export.snapshot.yaml:200]: There is no Action Erase with more than one step. We can add it as follows:

```suggestion
        "steps": [
          {
            "startTime": "2019-08-19T16:48:19.690458+00:00",
            "type": "StepRandom",
            "severity": "Info",
            "endTime": "2019-08-19T17:34:22.930562+00:00"
          },
          {
            "startTime": "2019-08-19T17:34:22.690458+00:00",
            "type": "StepZero",
            "severity": "Info",
            "endTime": "2019-08-19T18:34:22.930562+00:00"
          }
        ],
        "severity": "Info",
        "endTime": "2019-08-19T18:34:22.930959+00:00"
      }
```

**Review by JNadeu — COMMENTED** [review]
In general everything is fine, just fixing little things.

**Comment by cayop** [comment]
All changes is done

**Review by JNadeu — APPROVED** [review]

**Review by JNadeu — APPROVED** [review]

**Pull request merged**